src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+   Hello World
+</body>
+</html>
```

Comment:
git doesn't like it 😅 
<img width="203" alt="image" src="https://user-images.githubusercontent.com/55802630/172130083-00d265af-d9d7-46eb-a1a5-30af62bb4eeb.png">